lib/jsdom/level2/html.js

```diff
@@ -164,11 +166,46 @@ ResourceQueue.prototype = {
   }
 };
 
+function RequestManager(options) {
+  this.pool = options.pool;
```

Comment:
Hmm. I think it is weird to store them as member variables of the class if they don't actually relate to the class. So symbols on the document (or window) seems better.